validphys2/src/validphys/results.py

```diff
@@ -949,6 +949,26 @@ def fits_groups_chi2_table(
     return res
 
 
+@table
+def fits_groups_nsigma_table(
+    fits_name_with_covmat_label, fits_groups, fits_groups_chi2_data, per_point_data: bool = True
+):
+    df = fits_groups_chi2_table(
+        fits_name_with_covmat_label, fits_groups, fits_groups_chi2_data, per_point_data
+    )
```

Comment:
Can't you have this function directly as an input?

Reply:
No I don't think it works in this case as the `fits_groups_chi2_table` is not really used as a node of the graph in this case but as a normal function. See `dataspecs_groups_chi2_table` function for instance

Reply:
But for `dataspecs_` the input changes, here it doesn't you should have the entire input available when you call this function. Maybe you need a "notable" version without the decorator?

Reply:
Ah yes, you are right. Thanks for this, I should have added it now.